vector/src/main/java/im/vector/activity/VectorRoomCreationActivity.java

```diff
@@ -57,25 +57,27 @@
     // tags
     private static final String LOG_TAG = VectorRoomCreationActivity.class.getSimpleName();
 
+    private static final int INVITE_USER_REQUEST_CODE = 456;
+
     // participants list
     private static final String PARTICIPANTS_LIST = "PARTICIPANTS_LIST";
 
-    //
-    private static final int INVITE_USER_REQUEST_CODE = 456;
+    // add an extra to precise the type of filter we want to display contacts
+    public static final String EXTRA_INVITE_CONTACTS_FILTER = "EXTRA_INVITE_CONTACTS_FILTER";
 
     // add an extra to precise the type of mode we want to open the VectorRoomCreationActivity
     public static final String EXTRA_ROOM_CREATION_ACTIVITY_MODE = "EXTRA_ROOM_CREATION_ACTIVITY_MODE";
 
-    public enum RoomCreationModes { DIRECT_CHAT, DISCUSSION; }
+    // This enum is used to select a mode for a room creation
+    public enum RoomCreationModes { DIRECT_CHAT, DISCUSSION }
```

Comment:
We should add here a third mode: INVITE
This new mode should replace the current uses of DISCUSSION 
DISCUSSION may be conserved here as the default value